src/estimators/bloom_filters.py

```diff
@@ -419,6 +421,53 @@ def __call__(self, sketch_list):
         union, self._weights)]
 
 
+class FirstMomentGlobalNoiseEstimator(EstimatorBase):
+  """This estimator unions the input sketches, generate geometric noise,
+     appends that many fake buckets to the sketch union and calls the
+     FirstMomentEstimator.
```

Comment:
This only supports exp-adbf, right? Do you want to make this more general for all adbf families?

An alternative is to extend FirstMomentNoiseEstimator to take a global_estimator_noiser, which, if defined, will be added to #ones inside of FirstMomentNoiseEstimator.

Reply:
Yeah, I think that's a reasonable way to refactor. Will send an update.

Reply:
Another approach is to change the AnyDistributionBloomFilter, i.e., add an optional global noiser argument to the constructor, and create a property method for the number of one bits (the `x` in the exp and log bloom filter estimator, and the `n` in the geo bloom filter). In this way, we don't need to modify all the estimator methods. Please let me know if this makes sense.

Thanks Craig!

Reply:
PTAL. Refactored FirstMomentEstimator based on conversation yesterday.